states/huntsman/parked.py

```diff
@@ -5,4 +5,8 @@ def on_enter(event_data):
     pocs.say("I'm parked now. Phew.")
 
     pocs.say("Cleaning up for the night!")
-    pocs.next_state = 'housekeeping'
+
+    if pocs.is_dark and pocs.weather_is_bad:
```

Comment:
```suggestion
    if pocs.is_dark(horizon='flat') and pocs.is_weather_safe is False:
```

We don't have a `weather_is_bad`, I was just being lazy in my earlier comment (that's why it said it was psuedo-code).

Reply:
OK, thanks for the input, I'll change this